client/coral-framework/actions/items.js

```diff
@@ -220,8 +221,13 @@ export function postItem (item, type, id) {
 */
 
 export function postAction (item_id, item_type, action) {
-  return () => {
-    return coralApi(`/${item_type}/${item_id}/actions`, {method: 'POST', body: action});
+  return (dispatch, getState) => {
+    action._csrf = getState().auth.get('_csrf');
```

Comment:
Is it possible/advisable to add the _csrf token in the middleware so it is auto appended to all POST/PUT requests?